Tests/Cells/BaseBrickCellTests.swift

```diff
@@ -182,6 +182,44 @@ class BaseBrickCellTests: XCTestCase {
         XCTAssertNotNil(cell3?.contentView.viewWithTag(21))
         XCTAssertEqual(cell3?.contentView.subviews.first, backgroundView1)
     }
+    
+    func testBrickViewAppearance() {
+        brickView.registerBrickClass(LabelBrick.self)
+        
+        let testLabelBrick = LabelBrick("AppearanceTest", height: .Fixed(size: 100), text: "Appearance Test")
+        testLabelBrick.brickCellAppearanceDataSource = self
```

Comment:
Could you make an object that handles this instead of using `self`? You'll have some issues if you make more tests that need this